DevelopmentTools/HCALLMapDumper.cc

```diff
@@ -1239,3 +1239,21 @@ bool HCALLMapDumper::ifTableExistInDB(sqlite3 *db, std::string TableName)
   }
   return exist;
 }
+
+//Added by Yuan
+void HCALLMapDumper::printHOCalibMapObject(std::vector<HOCalib> myHOCalib)
```

Comment:
Please rename the function to be "printHOCalibLMapObject" to avoid future confliction. Also, change that in declaration and call.